src/components/HamburgerMenu/styles.module.css

```diff
@@ -20,40 +20,108 @@
   overflow-y: auto;
 
   @media (--xs-scr) {
-    visibility: visible;
+    display: block;
   }
 
   &.opened {
+    visibility: visible;
     transform: translateX(0);
+
+    a:focus {
+      outline-color: #fff;
+    }
   }
 }
 
 .toggleButton {
+  @mixin hover;
+
   position: fixed;
   display: none;
   visibility: hidden;
   z-index: 999;
-  right: 15px;
-  top: 25px;
+  right: 16px;
+  top: 12px;
   padding: 0;
   border: none;
+  width: 40px;
+  height: 40px;
+  align-items: center;
+  justify-content: center;
   background: transparent;
-  transition: 0.5s all;
+  outline: none;
+  transition: 0.5s all, opacity 0.2s, outline-offset 0s, outline-width 0s,
+    outline-color 0s;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &.opened {
+    outline-color: #fff;
+  }
 
   &.expanded {
-    top: 60px;
+    top: 47px;
   }
 
   @media (--xs-scr) {
-    display: block;
+    display: flex;
     visibility: visible;
   }
 }
 
+.logoRow {
+  display: flex;
+}
+
 .logo {
-  display: block;
-  width: 34px;
-  margin-top: 5px;
+  @mixin hover;
+  @mixin active;
+
+  width: 40px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &:focus path {
+    fill: var(--color-orange);
+  }
+}
+
+.company {
+  @mixin active;
+  @mixin hover;
+
+  font-size: 15px;
+  line-height: 20px;
+  font-weight: 500;
+  letter-spacing: 0.02em;
```

Comment:
Let's get rid of all letter-spacing in the header elements (0.2em → 0), the reason for this was to improve readability, but it looks a bit weird.